cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/AbstractStreamFileAdmin.java

```diff
@@ -273,9 +279,40 @@ public void create(String name, @Nullable Properties props) throws Exception {
     StreamConfig config = new StreamConfig(name, partitionDuration, indexInterval, ttl, streamLocation, null);
     saveConfig(config);
 
+    // Create the notification feeds linked to that stream
+    createStreamFeeds(name);
+
     streamCoordinatorClient.streamCreated(name);
   }
 
+  private void createStreamFeeds(String stream) {
+    // TODO use accountID as namespace?
+    try {
+      NotificationFeed streamFeed = new NotificationFeed.Builder()
+        .setNamespace("default")
+        .setCategory(Constants.Notification.Stream.STREAM_FEED_CATEGORY)
+        .setName(stream)
+        .setDescription(String.format("Size updates feed for Stream %s every %dMB",
+                                      stream, Constants.Notification.Stream.DEFAULT_DATA_THRESHOLD))
+        .build();
+      notificationFeedManager.createFeed(streamFeed);
+    } catch (NotificationFeedException e) {
+      LOG.error("Cannot create feed for Stream {}", stream, e);
```

Comment:
I'm not sure what should be done here, if the feeds can't be created. We should talk about it.